src/equations/linear_scalar_advection_1d.jl

```diff
@@ -145,6 +145,31 @@ end
 end
 
 
+# Essentially first order upwind, see e.g.
+# https://math.stackexchange.com/a/4355076/805029
+function flux_godunov(u_ll, u_rr, orientation::Int, equation::LinearScalarAdvectionEquation1D)
+  u_L = u_ll[1]
+  u_R = u_rr[1]
+
+  if equation.advection_velocity[orientation] >= 0
+    return SVector(equation.advection_velocity[orientation] * u_L)
+  else 
+    return SVector(equation.advection_velocity[orientation] * u_R)
+  end
```

Comment:
```suggestion

  v_normal = equation.advection_velocity[orientation] 
  if v_normal >= 0
    return SVector(v_normal * u_L)
  else 
    return SVector(v_normal* u_R)
  end
```
This avoids the possibility of needing to look up the value multiple times and clarifies the intention. Could you please adapt the others as well?

Reply:
Addressed in commit [5fb9499e88df5a9ce3e4f562d598f8c7b18d0ff7](https://github.com/trixi-framework/Trixi.jl/commit/5fb9499e88df5a9ce3e4f562d598f8c7b18d0ff7).